bin/cml/check/create.js

```diff
@@ -10,7 +10,11 @@ exports.handler = async (opts) => {
   await cml.checkCreate({ ...opts, report });
 };
 
-exports.builder = (yargs) => yargs.env('CML_CHECK').options(exports.options);
+exports.builder = (yargs) =>
+  yargs
+    .env('CML_CHECK')
+    .option('options', { default: exports.options, hidden: true })
```

Comment:
This is a gross hack, because otherwise there is no way[^1] of accessing the options object from a middleware.

[^1]: None that I know of